examples/user_guide/04-Style_Mapping.ipynb

```diff
@@ -680,9 +682,22 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
+   "display_name": "Python 2",
```

Comment:
You really should upgrade to Python3 😄 

Also ordinarily we clear this metadata because it can confuse other peoples notebook sessions. I'll push to your PR to remove it.